wasm/src/primitives/bexter.rs

```diff
@@ -91,3 +91,11 @@ impl Deref for BexterWrapper {
         &self.0
     }
 }
+
+impl Wrap<Bexter> for BexterWrapper {
+    type Wrapper = BexterWrapper;
+
+    fn wrap(verfer: &Bexter) -> Self::Wrapper {
```

Comment:
Why not name all these `verfer` variables `matter` or explicitly the correct name?

Reply:
Oops, just a mustype. I'll fix.